app/views/users/show.html.erb

```diff
@@ -2,16 +2,17 @@
   <!-- pseudoflash because putting it in a real one was making for some ugly code -->
   <div class="flash notice" id="first-login-help-banner">
     <p>
-      Hi! It looks like you've just logged into the archive for the first time. 
-      <%= link_to_modal ts("Learn some tips and tricks"), :for => first_login_help_path, :title => "First login help" %> 
-      or dismiss this message permanently.
+      <%= t(".login_banner.welcome_text", help_link: link_to_modal(t(".login_banner.help_link"), :for => first_login_help_path, :title => t(".login_banner.help_title"))).html_safe %>
```

Comment:
Would you mind updating the syntax where possible as long as you're here, so it's `for: first_login_help_path`, etc?